src/Asset/TagRenderer.php

```diff
@@ -54,6 +58,10 @@ public function renderWebpackScriptTags(string $entryName, string $packageName =
                 $attributes['integrity'] = $integrityHashes[$filename];
             }
 
+            if (isset($this->defaultAttributes['crossorigin']) && false !== $this->defaultAttributes['crossorigin']) {
```

Comment:
this does not make sense. If you take an array `$defaultAttributes` in the renderer, you should merge them in attributes, not require implementing support for them one by one.

Reply:
ok thats probably beyond my symfony skill level - sorry. Happy for you to propose the code, show me an example, or take over this?

Reply:
Well, instead of `$attributes = [ 'src' => ... ]` at the beginning, make it like that:

```php
$attributes = $this->defaultAttributes;
$attributes['src'] = ...
```

there is nothing specific to Symfony here.

Reply:
note that this snippet assumes that `$this->defaultAttributes` contains only attributes which should be applied, not keys with `false` values when omitting them.

Reply:
ah I thought you meant something with an options resolver :) 

ok I'll give it another go :)

Reply:
ok committed a change to incorporate your feedback. ok?